cmd/kops/create_cluster.go

```diff
@@ -936,15 +930,24 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 					MaxRetries: fi.Int(3),
 				},
 			}
+			if c.APILoadBalancerType != "" {
+				// Set defaults for an openstack LB
```

Comment:
I think maybe you _could_ move this down to where we make the choice between DNS vs Load Balancers later in the file (around line 1190).  It would mean checking again to see if `cloud == "openstack"`, but that's fine...